neural_speed/convert/convert_llama.py

```diff
@@ -178,6 +181,11 @@ def loadHFTransformerJson(model: 'LazyModel', config_path: Path) -> 'Params':
         ffn_hidden_size = config["intermediate_size"]
         rms_norm_eps = config["rms_norm_eps"]
         rope_theta = config["rope_theta"] if "rope_theta" in config else 10000
+        rope_scale = 1
+        if config["rope_scaling"]:
```

Comment:
please check whether "rope_scaling"  in config
```suggestion
        if "rope_scaling" in config and config["rope_scaling"] is not None:
```